tests/README.md

```diff
@@ -33,7 +33,7 @@
 
 1. Run `make dm_integration_test_build` to generate DM related binary for integration test
 
-2. Setup two MySQL servers with binlog enabled first and set `GTID_MODE=ON`, export proper environment variables. Variables and their default values are showed in following table.
+2. Setup two MySQL servers (<5.8.0) with binlog enabled first and set `GTID_MODE=ON`, export proper environment variables. Variables and their default values are showed in following table.
```

Comment:
Maybe <8.0 will be better? Actually we currently not support mysql 8.0 both test and production environment.

Reply:
noticed current dependence of `tidb-tools@v4.0.2-0.20200622075412-c409811814de+incompatible/pkg/check/mysql_server.go`

```
// SupportedVersion defines the MySQL/MariaDB version that DM/syncer supports
// * 5.6.0 <= MySQL Version < 5.8.0
// * 10.1.2 <= Mariadb Version
var SupportedVersion = map[string]struct {
	Min MySQLVersion
	Max MySQLVersion
}{
	"mysql": {
		MySQLVersion{5, 6, 0},
		MySQLVersion{5, 8, 0},
	},
	"mariadb": {
		MySQLVersion{10, 1, 2},
		MaxVersion,
	},
}
```

This PR is mainly caused by long error log which is hard to locate when I run integration test 😢

Reply:
there is no MySQL's version named `5.8.0`, how about changing it to `<=5.7`?

Reply:
> there is no MySQL's version named `5.8.0`, how about changing it to `<=5.7`?

OK, going to change it to `5.6~5.7`